jck/jck.mk

```diff
@@ -68,8 +68,14 @@ endif
 OTHER_OPTS=
 # if JDK_IMPL is openj9 or ibm
 ifneq ($(filter openj9 ibm, $(JDK_IMPL)),)
- OTHER_OPTS=-Xtrace:maximal=all{level2}
- export CONFIG_ALT_PATH:=$(JCK_ROOT)$(D)config$(D)default
+  OTHER_OPTS=" -Xtrace:maximal=all{level2} -Xfuture "
+  export CONFIG_ALT_PATH:=$(JCK_ROOT)$(D)config$(D)default
+  ifeq ($(OS), OS/390)
```

Comment:
Also, this condition doesn't detect z/OS platform correctly (checked in a grinder). I suggest we use the following: 

```
ifneq (,$(findstring zos,$(SPEC)))
```